easybuild/easyconfigs/x/XCFun/XCFun-2.1.0-foss-2020a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'XCFun'
+version = '2.1.0'
+
+homepage = 'xcfun.readthedocs.io'
+description = """Arbitrary order exchange-correlation functional library"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/dftlibs/xcfun/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['64aac8c933cc129ce6326f3827e342abfd10b94ea4a302aaca9f10d342ad7656']
+
+builddependencies = [
+    ('CMake', '3.16.4', '', ('GCCcore', '9.3.0'))
+]
+
```

Comment:
No more python dep?

Reply:
Well, XCFun is a C++ library with an optional Python interface, and this particular build would be without Python support. I tried first to make the general `XCFun-2.1.0-foss-2020a-Python-3.8.2.eb` version, but it pulled in some more dependencies which I don't really need (but PySCF does). Do you think it's worth having a separate Python-free version like this, or should we only provide the general version _with_ Python support?

Reply:
Typically I would say, do opt in, unless there is some reason not to. 

If we don't want to opt it, then it might be possible to move this up the hierarchy.

Reply:
Not sure what you mean with moving up the hierarchy?

I discussed briefly with XCFun maintainer @robertodr. The Python interface is not widely used, so we think a separate recipe without Python support is worth while.

Reply:
I agree that we can merge this easyconfig without Python as the name of the easyconfig will be different in any case. If anybody needs support for Python it can be added later on.

@stigrj  @Micket was talking about changing the toolchain of the easyconfig. Since XCFun does not use BLAS/LAPACK, FFTW or MPI, then it does not need the `foss` toolchain. It can be moved to `GCC-10.2.0`, making it more compatible with other easyconfigs.

Reply:
Thanks for the input @lexming! I switched to `GCCcore-9.3.0`, I don't think 10 is available yet on my clusters :slightly_smiling_face: